hbase-server/src/main/java/org/apache/hadoop/hbase/master/procedure/InitMetaProcedure.java

```diff
@@ -61,9 +64,14 @@ public class InitMetaProcedure extends AbstractStateMachineTableProcedure<InitMe
 
   private RetryCounter retryCounter;
 
+  protected final Configuration conf;
+  public InitMetaProcedure(Configuration conf) {
+    this.conf = conf;
+  }
+
```

Comment:
You probably don't need this constructor at all if you return `TableName.META_TABLE_NAME` in `getTableName()` method. Actually don't need to modify this class at all.